docs/kubernetes/how-to-migrate-from-k8s-alpha-to-terraform/index.md

```diff
@@ -0,0 +1,257 @@
+          default = 5
+        }
+
+1.  Once your edit is made, move back to the `/.k8s-alpha-linode/clustername` directory and apply your changes with Terraform:
+
+        terraform apply
+
+    {{< note >}}
+You may need to use the original Terraform version used to deploy the cluster (either Terraform 0.11.X or Terraform 0.12.X). If you do not, you will see syntax errors.
+{{< /note >}}
+
+1.  Once this is completed, you'll see a prompt reviewing your changes and asking if you would like to accept them. To proceed, type `yes` and Terraform will proceed to make the changes. This process may take a few moments.
+
+    {{< note >}}
+When prompted, you may notice that one item in your plan is marked as `destroy`. This is generally a "null resource" or a local script execution and is not indicative of an unintended change.
+{{</ note >}}
+
+1.  After Terraform has finished, your cluster will be resized. To confirm, enter the following command to list all nodes in your cluster, replacing the string `mycluster` with the name of the cluster you edited:
+
+        kubectl --kubeconfig=mycluster.conf get nodes
+
+1.  The output will then list an entry for each node:
+
+    {{< output >}}
+kubectl --kubeconfig=mycluster.conf get nodes
+NAME             	STATUS   ROLES	  AGE 	  VERSION
+mycluster-master-1      Ready	 master   21m 	  v1.13.6
+mycluster-node-1 	Ready	 &lt;none&gt;   18m 	  v1.13.6
+mycluster-node-2 	Ready	 &lt;none&gt;   18m 	  v1.13.6
+mycluster-node-3 	Ready	 &lt;none&gt;   18m 	  v1.13.6
+mycluster-node-4 	Ready	 &lt;none&gt;   4m26s   v1.13.6
+mycluster-node-5 	Ready	 &lt;none&gt;   4m52s   v1.13.6
+{{< /output >}}
+
+### Upgrade a Cluster
+
+You may have noticed that the Terraform module, `cluster.tf`, refers to a specific branch or git commit hash referencing a remote Terraform module on GitHub. The following section will outline how to upgrade your cluster to the latest version of this module.
+
+For example, your `source` variable may have a value that points to the git branch ref `for-cli`. To perform an upgrade this must point to the latest commit history hash.
+
+1.  Visit the branch of the Terraform module you're using on [Github](https://github.com/linode/terraform-linode-k8s/commits/for-cli). Note the commit history and copy the latest hash by clicking on the clipboard next to the hash of the most recent commit. At the time of this writing, the most recent hash is as follows:
+
+    {{< output >}}
+5e68ff7beee9c36aa4a4f5599f3973f753b1cd9e
+{{< /output >}}
+
+1.  Edit `cluster.tf` to prepare for the upgrade. Update the following section using the hash you copied to appear as follows:
+
+    {{< file "cluster.tf" >}}
+source = "git::https://github.com/linode/terraform-linode-k8s.git?ref=5e68ff7beee9c36aa4a4f5599f3973f753b1cd9e"
+{{< /file >}}
+
+1.  To apply these changes, re-init the modules by running the following command:
+
+        terraform init
+
+1.  Once this has completed, run terraform apply:
```

Comment:
Once this has completed, apply your changes with the following command: (just trying to remove the command from the sentence)

Reply:
updated